### ISSUE TITLE
Simple start_genes question

### ISSUE THREAD

**Issue by dsladevt** [rrwick/Unicycler#68]
Ryan,

If I'm using the --start_genes option, and I have a file start_genes.fasta in the main folder, do I just call it as 1) --start_genes start_genes.fasta, or 2) --start_genes START_GENES, where START_GENES is a folder and the start_genes.fasta is in it, or 3) is it enough just to put --start_genes in the command and have the start_genes.fasta file in the folder?

I got an error with the first two.

Thanks in advance.

**Comment by dsladevt** [comment]
I think after reading the documentation it appears that as long as I have start_genes.fasta in the folder I don't have to put anything in the command line?

**Comment by rrwick** [comment]
By default, Unicycler uses the `start_genes.fasta` file that's included in its source directory ([this one](https://github.com/rrwick/Unicycler/blob/master/unicycler/gene_data/start_genes.fasta)). You could potentially overwrite that file, in which case Unicycler would use it by default, but that's a somewhat hacky solution.

It would be better to use the `--start_genes` option and point it to the file you want to use. For example:
```
unicycler (other options) --start_genes path/to/my_favourite_start_genes.fasta
```

If when you say 'main folder' you mean the directory you're running Unicycler from, then your option 1 works. But you can put the fasta file anywhere - just use the appropriate path with the `--start_genes` option.

Looking at the command line help I can see that the help text for `--start_genes` is a bit ambiguous. I'll try to reword it to be clearer.

Hope that all makes sense. I'll close the issue now, but please let me know if anything's unclear!

Ryan

**Comment by dsladevt** [comment]
Ryan,

I got it to work by giving it the complete path and it looks great.  Thanks for the help.
